pallets/capacity/README.md

```diff
@@ -0,0 +1,59 @@
+# Capacity Pallet
+
+Manages the staking and balances for Capacity, an alternative payment system on Frequency.
+
+## Summary
+
+Capacity is an alternative payment system for a subset of calls that renews each Epoch.
```

Comment:
Two questions here that a new user may not know the answer to:
- what is the alternative if we're not using capacity?
- what is the subset of calls?

Reply:
I'm trying to avoid too much discussion of what Capacity is. If you have gotten all the way to these docs, I hope you are on docs.frequency.xyz and you can check the glossary...

> Subset of calls

My math degree showing through. Ideas for better ways to say this?

Reply:
```suggestion
Capacity is a renewable payment resource (renewable each [Epoch](#capacity-epoch)) for certain eligible transactions, that provides an alternative to paying directly with `FRQCY` tokens.
```

Reply:
Another alternative:
```suggestion
Capacity is an alternative to paying with FRQCY for a selection of calls. Generated Capacity for a Provider renews each Epoch.
```

Reply:
Revised the section